mmhuman3d/utils/ffmpeg_utils.py

```diff
@@ -1056,10 +1063,11 @@ def slice_video(input_path: str,
     info = vid_info_reader(input_path)
     num_frames = int(info['nb_frames'])
     start = (min(start, num_frames - 1) + num_frames) % num_frames
-    end = (min(end, num_frames - 1) + num_frames) % num_frames
+    end = (min(end, num_frames - 1) +
+           num_frames) % num_frames + 1 if end is not None else num_frames
```

Comment:
Similar as above.

Reply:
in ffmpeg, the end_frame must be positive